security/src/main/java/uk/ac/ox/ndph/mts/security/authentication/SecurityContextUtil.java

```diff
@@ -22,6 +22,12 @@ public String getUserId() {
         return userPrincipal.getClaim("oid").toString();
     }
 
+    public boolean isInitService() {
```

Comment:
Can you rename the function to reflect what it does, something like isInIdentityProviderRole... (and add doc to explain the difference of FHIR roles)?